api/src/main/java/io/opencensus/trace/SpanBuilder.java

```diff
@@ -133,12 +133,22 @@
    * Sets the option {@link Span.Options#RECORD_EVENTS} for the newly created {@code Span}. If not
    * called, the implementation will provide a default.
    *
-   * @param recordEvents New value determining if this {@code Span} should have events recorded.
+   * @param recordEvents new value determining if this {@code Span} should have events recorded.
    * @return this.
    * @since 0.5
    */
   public abstract SpanBuilder setRecordEvents(boolean recordEvents);
 
+  /**
+   * Sets the {@link Span.Kind} for the newly created {@code Span}. If not called, the
+   * implementation will provide a default.
```

Comment:
Can the default be null?  It looks like this implementation doesn't set a non-null default.

Reply:
Yes, null is a valid option. we can do unspecified if really is important but I think null is fine.